examples/deployments/EKS/Deploy_VLLM_example.md

```
@@ -4,7 +4,7 @@
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)